### PR TITLE
Check if runtime is Python2

### PR DESCRIPTION
**Pull request by andriisoldatenko**
Add ability to check python runtime

### CODE REVIEW

**Comment by jamesls** [comment]
I think this can still work if I just skip the `pip install -e` on python3.  I'll update this before merging.

**Comment by andriisoldatenko** [comment]
will update.

**Comment by jamesls** [comment]
@andriisoldatenko Thanks!  To get the mypy stuff working, we can do something like this for now:

```
  - if [[ $TRAVIS_PYTHON_VERSION != 3.* ]]; then pip install -e .
  - if [[ $TRAVIS_PYTHON_VERSION != 3.* ]]; then travis_retry pip install -r requirements-dev.txt
  - if [[ $TRAVIS_PYTHON_VERSION == 3.* ]]; then pip install mypy-lang==0.4.5; fi
```

Probably worth moving out to a separate shell script if this gets any more complicated, unless there's a better way to do this.

**Comment by andriisoldatenko** [comment]
@jamesls  @dstufft thanks guys for suggestions.

File: setup.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 from setuptools import setup, find_packages
-
+import sys
```
Comment (dstufft) [inline setup.py]: This import is harmless but unused.

**Comment by jamesls** [comment]
Thanks!

**Pull request merged**